extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -400,12 +392,62 @@ function getJsonItem( $titleName, $ns, $pageId ) {
 	 * @return bool
 	 */
 	static function onCuratedContentSave() {
-		self::purgeMethod( 'getList' );
+		global $wgServer;
+
+		$content = F::app()->wg->WikiaCuratedContent;
+
+		( new SquidUpdate( array_unique( array_reduce(
+			$content,
+			function ( $urls, $item ) use ( $wgServer ) {
+				if ( $item[ 'title' ] !== '' && empty( $item[ 'featured' ] ) ) {
+					// Purge section URLs using urlencode() (standard for MediaWiki), which uses implements RFC 1738
+					// https://tools.ietf.org/html/rfc1738#section-2.2 - spaces encoded as `+`.
+					// iOS apps use this variant.
+					$urls[ ] = self::getUrl( 'getList' ) . '&section=' . urlencode( $item[ 'title' ] );
```

Comment:
Hard to tell, it's not a definition, but it's also not strictly indexing.  https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax
I would rather go without spaces inside, as this construct is closer to indexing. I wrote it this way to be consistent with the rest of the file.